src/control/server/harness.go

```diff
@@ -39,21 +40,23 @@ import (
 // IOServerHarness is responsible for managing IOServer instances
 type IOServerHarness struct {
 	sync.RWMutex
-	log         logging.Logger
-	instances   []*IOServerInstance
-	started     uint32
-	restartable uint32
-	restart     chan struct{}
-	errChan     chan error
+	log            logging.Logger
+	instances      []*IOServerInstance
+	started        uint32
+	restartable    uint32
+	restart        chan struct{}
+	errChan        chan error
+	rankReqTimeout time.Duration
 }
 
 // NewHarness returns an initialized *IOServerHarness
 func NewIOServerHarness(log logging.Logger) *IOServerHarness {
 	return &IOServerHarness{
-		log:       log,
-		instances: make([]*IOServerInstance, 0, maxIoServers),
-		restart:   make(chan struct{}, 1),
-		errChan:   make(chan error, maxIoServers),
+		log:            log,
+		instances:      make([]*IOServerInstance, 0, maxIOServers),
+		restart:        make(chan struct{}, 1),
+		errChan:        make(chan error, maxIOServers),
+		rankReqTimeout: 3 * time.Second,
```

Comment:
If I understand correctly, the intent behind adding this to the harness is to allow the timeout to be configurable for testing. That's fine, but it would be better for this value to be defined in a const, e.g. `defaultRequestTimeout`.